src/core.jl

```diff
@@ -136,7 +159,38 @@ end
 #####  Kernel Naive Bayes       #####
 #####################################
 
+"""
+    fit(m::HybridNB, f_c::Vector{Vector{Float64}}, f_d::Vector{Vector{Int64}}, labels::Vector{Int64})
+
+Train NB model with discrete and continuous features by estimating P(x⃗|c)
+"""
+function fit{C, T<: AbstractFloat, U<:Int}(model::HybridNB, continuous_features::Vector{Vector{T}}, discrete_features::Vector{Vector{U}}, labels::Vector{C})
+    for class in model.classes
+        inds = find(labels .== class)
+        for (j, feature) in enumerate(continuous_features)
+            model.c_kdes[class][j] = InterpKDE(kde(feature[inds]), eps(Float64), InterpLinear)
+        end
+        for (j, feature) in enumerate(discrete_features)
+            model.c_discrete[class][j] = ePDF(feature[inds])
+        end
+    end
+    return model
+end
+
+
+"""
+    fit(m::HybridNB, f_c::Matrix{Float64}, labels::Vector{Int64})
+
+Train NB model with continuous features only
+"""
+function fit{C, T<: AbstractFloat}(model::HybridNB, continuous_features::Matrix{T}, labels::Vector{C})
+    discrete_features = Vector{Vector{Int64}}()
+    return fit(model, restructure_matrix(continuous_features), discrete_features, labels)
+end
+
+
 function fit{C}(m::KernelNB, X::Matrix, y::Vector{C})
+    warn("fit method for KernelNB is deprecated. Use HybridNB instead.")
```

Comment:
Since we haven't released any version with `KernelNB`, I think it's safe to remove it at all.